packages/core/src/env-set/index.ts

```diff
@@ -63,7 +63,7 @@ export class EnvSet {
     return this.#oidc;
   }
 
-  async load() {
+  async load(customEndpoint?: string) {
```

Comment:
We can just use `customDomain` for this? Since we can say "endpoint is calculated from a custom domain"